tests/iio_stresstest.c

```diff
@@ -152,12 +152,7 @@ static struct iio_device * get_device(const struct iio_context *ctx,
 	return NULL;
 }
 
-enum backend {
```

Comment:
i like enums that are each on it's own line

Reply:
BreakBeforeBraces should be set to Linux

Reply:
Agree - reading more doc says:

clang-format contracts enums to a single line if all elements fit on one line. This conserves spaces and usually does not decrease readability. There is no style option, but you can override this by either adding a line comment somewhere or by adding a trailing comma after the last enumerator,

so I will do that manually.